robot.py

```diff
@@ -129,6 +129,12 @@ def teleopInit(self):
         self.pistons_activated = False
         self.forward = 0
 
+        #Vision sensor
+        self.visionSensor = VisionSensor('127.0.0.1', 8812)
```

Comment:
Can we figure out how to determine if we are "doing simulator" and have this use the preferred IP if we are not?    I don't we want to commit this loopback into master.